Sources/TemplateKit/AST/TemplateSyntax.swift

```diff
@@ -0,0 +1,106 @@
+import Foundation
```

Comment:
This file misses comments entirely. It's quite simple to understand in many places, though